content/blog/status-update-27-20.md

```diff
@@ -0,0 +1,94 @@
+- More prominent promotion of AsyncAPI maintained tools, like [the Generator](https://deploy-preview-108--asyncapi-website.netlify.app/generator)
+- A more unobstructed view of the [Blog](https://deploy-preview-108--asyncapi-website.netlify.app/blog) section
+- Right-hand side page navigation with scroll spy on [documentation view](https://deploy-preview-108--asyncapi-website.netlify.app/docs/specifications/2.0.0).
+
+There are many other significant changes. You'll see them immediately on the main landing page.
+
+## The Highlight of Interesting Discussions
+
+There are some interesting discussions/topics where it would be great to hear your opinion.
+
+### Circular References
+
+How would you expect those are handled by the Parser and also by the Generator? Should such references be ignored or resolved, and if resolved, then how? Should generated docs indicate circular reference or ignore this info as irrelevant? These are all the questions we face now, and it would be awesome to get your input [here](https://github.com/asyncapi/parser-js/issues/83).
+
+### React Wrapper Redoc style
+
+AsyncAPI Initiative provides two ways of generating docs from AsyncAPI specification:
+
+- [HTML template](https://github.com/asyncapi/html-template) for the AsyncAPI generator. The classical server-side generation you perform in your CI pipelines or in an application with Node.js server.
+- [React component](https://github.com/asyncapi/asyncapi-react/) that you can use for dynamic docs rendering client-side. This one is great, but only if you want to use React in your application.
+
+We are thinking about how we could consolidate both into one project that is maintained just once. In the meantime, the community created [this wrapper](https://github.com/jfveronelli/instant-asyncapi-doc) for our React component inspired by ReDoc approach to OpenAPI. It would be great to know what you think about this and what are your expectations toward AsyncAPI Initiative. Join the [discussion](https://github.com/asyncapi/asyncapi/issues/391).
+
+### React for Template Engine for the Generator
+
+We are getting a bit tired of Nunjucks and how hard it is to debug and read the templates. Fran came up with an idea to use React as a template engine for our docs and code generator. He got inspired by the emerging popularity of solutions like Gatsby or Next.js (supporting pre-rendering of React components). Before you say out loud "you are crazy" join the discussion [here](https://github.com/asyncapi/generator/issues/377).
+
+## AsyncAPI Special Interest Group (SIG) open meeting
+
+The last meeting took place on Tuesday, 23rd of June, 4PM UTC. Meeting notes are available [here](https://github.com/asyncapi/asyncapi/issues/394).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/LDanOihP9i4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Would you mind changing the style? Look at this one: https://deploy-preview-113--asyncapi-website.netlify.app/blog/status-update-week-14-2019/